Model/IdealPaymentMethod.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use Wirecard\PaymentSdk\Entity\IdealBic;
+use Wirecard\PaymentSdk\Transaction\IdealTransaction;
+use Wirecard\PaymentSdk\Transaction\Transaction;
+
+use Wirecard\Oxid\Core\Helper;
+
+use OxidEsales\Eshop\Core\Registry;
+
+//use ReflectionClass;
+
+/**
+ * Payment method implementation for iDEAL
+ *
+ * @since 1.2.0
+ */
+class IdealPaymentMethod extends PaymentMethod
+{
+    /**
+     * @inheritdoc
+     *
+     * @since 1.2.0
+     */
+    protected static $_sName = "ideal";
+
+    /**
+     *
+     * @since 1.2.0
+     */
+    private static $_aBankOptions = [];
```

Comment:
no need for `static` if you always assign it in the constructor.

Reply:
:+1: